qt/widgets/common/src/QtPropertyBrowser/qteditorfactory.cpp

```diff
@@ -1547,6 +1547,8 @@ Q_GLOBAL_STATIC(QtCursorDatabase, cursorDatabase)
 
 QtCursorEditorFactoryPrivate::QtCursorEditorFactoryPrivate() : m_updatingEnum(false) {}
 
+// to allow updating the enum property, property cannot be const
```

Comment:
Out of interest why is this the case, does `QtProperty *enumProp = m_propertyToEnum.value(property);` mutate `property`?

Reply:
Yes,  the QtProperty is the key for the QMap, so the value for the key can change.